keras/src/backend/openvino/numpy.py

```diff
@@ -1291,7 +1291,96 @@ def reciprocal(x):
 
 
 def repeat(x, repeats, axis=None):
-    raise NotImplementedError("`repeat` is not supported with openvino backend")
+    x = get_ov_output(x)
```

Comment:
I think you can use `Tile` operation from our opset. It should be much easier to express `np.repeats`